### DISCUSSION TITLE
How to change field names?

### DISCUSSION THREAD

**Discussion by dwery** [flant/grafana-statusmap#162] [Q&A] [answered]
I'm using this plugin with the AWS Timestream data source to show a door status over time.
That datasource produces dataframes where the column name is something like "_col1 DeviceName", which would end on the panel.

In the standard graph I'd use the Field options to change it:
![image](https://user-images.githubusercontent.com/249355/100535001-e7148000-31c9-11eb-9231-f354a04aac73.png)

Is there a similar option for Statusmap?

**Comment by diafour** [comment, marked as answer]
Sorry, the statusmap doesn't support [Data frames](https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/) introduced in Grafana 7.0+. Only "time series" format is supported. This format lacks information and has only value defined in "Legend" (or "Alias") field in Query editor. Transformations and Data frames should be supported in the next releases.
